notebooks/feature_engineering/raw/ex1.ipynb

```diff
@@ -0,0 +1,412 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 1: Building a Baseline Model\n",
+    "\n",
+    "In this exercise, you will develop a baseline model for predicting if a customer will buy the app after clicking through from an ad. With this baseline model, you'll be able to see how your feature engineering and selection efforts improve the model's performance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up code checking\n",
+    "from learntools.core import binder\n",
+    "binder.bind(globals())\n",
+    "from learntools.feature_engineering.ex1 import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "click_data = pd.read_csv('../input/feature-engineering-data/train_sample.csv', \n",
+    "                         parse_dates=['click_time'])\n",
+    "click_data.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Baseline Model\n",
+    "\n",
+    "The first thing you need to do is construct a baseline model. All new features, processing, encodings, and feature selection should improve upon this baseline model. First you need to do a bit of feature engineering before training the model itself.\n",
+    "\n",
+    "### 1) Exercise: Features from timestamps\n",
+    "From the timestamps, create features for the day, hour, minute and second. Store these as new integer columns `day`, `hour`, `minute`, and `second` and a new DataFrame `clicks`."
+   ]
```

Comment:
It's kind of a drag to go looking up the exact syntax for this, when it feels like we're just building the baseline.

It'd be easier (while no less conceptually interesting) if you give them an example of the syntax and have them fill out the rest. Something like
clicks = pd.DataFrame({
'day': click_data.click_time.dt.day,
'hour': ____,
'minute': ____,
'second': ____
})
```

At that point it's really easy, which I think is a shortcoming. But I don't think I gained a lot by looking up the `dt` syntax.